typescript/infra/config/environments/mainnet3/funding.ts

```diff
@@ -119,7 +119,7 @@ export const keyFunderConfig: KeyFunderConfig<
     taiko: '0.2',
     tangle: '2',
     telos: '100',
-    treasure: '800',
+    treasure: '900',
```

Comment:
why the increase?

Reply:
So I hadn't realised that key funder will only fund up to the desired balance when the balance is less than half the desired balance (400), this takes us close the alert threshold